RRTStarMotionPlanner:
  footprint: [[0.5, 0.2], [0.5, -0.2], [-0.2, -0.2], [-0.2, 0.2]]
  planner:
    weight_d: 1.0
    weight_q: 1.0
    weight_c: 0.02
    weight_upstream: 0.1
    weight_dtc: 0.02
    weight_intensity: 0.2
    path_resolution: 0.1
    publish_viz_markers: true
  vehicle:
    inflation_radius: 0.1
    turning_radius: 0.5
    max_vehicle_speed: 1.0
  robot_id: $(arg robot_id)
